packages/apps/offers-notification/src/event-to-offers-notification.ts

```diff
@@ -0,0 +1,46 @@
+import type { ApiEventHandler } from '@cloudcommerce/firebase/lib/helpers/pubsub';
+import logger from 'firebase-functions/lib/logger';
```

Comment:
```suggestion
import logger from 'firebase-functions/logger';
```

Teve uma atualização major do `firebase-functions` e não tem mais esse export 🙃

Reply:
https://github.com/ecomplus/cloud-commerce/commit/1728d071dc1a00376a69619c8277d4f6a813da06